utils/api/ghost.js

```diff
@@ -19,6 +19,30 @@ export default class Ghost {
         return await ghost.site.read();
     }
 
+    /**
+     * Returns the number of 'active' newsletters.
+     *
+     * @returns {Promise<number>} The number of active newsletters.
+     */
+    async newslettersCount() {
+        const newsletters = await this.newsletters();
+        return newsletters.meta?.pagination?.total ?? 0;
```

Comment:
Default value should be `1` due to this - https://github.com/ItzNotABug/ghosler/blob/a507c216b53cfe14e2cb1d8b36b47d6b3fd3dfc0/routes/published.js#L35